benchmark/httpapi.py

```diff
@@ -277,18 +433,23 @@ def main(reactor, args):
         sys.stderr.write(options.getUsage())
         raise SystemExit(1)
 
+    def get_backend_from_options(options):
+        backend = _BACKENDS[options['backend']]
```

Comment:
:question: Something nicer than a KeyError on a typo would be nice, but incredibly low priority.